apps/dotcom/src/components/DocumentName/DocumentName.tsx

```diff
@@ -258,18 +260,20 @@ const DocumentNameEditor = track(function DocumentNameEditor({
 	const handleKeydownCapture = useCallback(
 		(e: KeyboardEvent) => {
 			if (e.key === 'Enter') {
-				e.preventDefault()
+				preventDefault(e)
 				// blur triggers save
 				inputRef.current?.blur()
 			} else if (e.key === 'Escape') {
-				e.preventDefault()
+				preventDefault(e)
+				stopEventPropagation(e)
 				// revert to original name instantly so that when we blur we don't
 				// trigger a save with the new one
 				setState((prev) => ({ ...prev, name: null }))
 				inputRef.current?.blur()
+				editor.getContainer().focus()
```

Comment:
(i've been staring at focus too long). i don't think we need this line - looks like just doing stopEventPropagation will do the trick 👍

Reply:
ahh interesting interesting, didn't realize that! can you add a note that that's what the focus does? definitely we'll forget why we put that there later :)